shared/src/store/action-reducers/transfer-point.ts

```diff
@@ -236,7 +290,7 @@ function estimateDuration(startPosition: Position, targetPosition: Position) {
     const overheadSummand = 10 * 1000;
     // In meters per second
     // On average an RTW drives 11.5 m/s (41.3 km/h https://www.leitstelle-lausitz.de/leitstelle/haeufig-gestellte-fragen/#:~:text=Wie%20viel%20schneller%20ist%20ein,30%2C4%20km%2Fh.)
-    // Be aware that this could be significantly off for longer distances due to, e.g., the use of the Autobahn.
+    // Be aware that this could be significantly off for longer distances due to, e.g., the use of an Autobahn.
```

Comment:
I think there is a difference between the German and English use of this word. In English you mostly mean *the* federal controlled-access highway system in Germany. Whereas in German "Autobahn" is only the type of a street. You can check the beginnings of the German and English Wikipedia articles for reference: https://en.wikipedia.org/wiki/Autobahn https://de.wikipedia.org/wiki/Autobahn